.github/workflows/check-build-installers.yml

```diff
@@ -0,0 +1,27 @@
+name: "[Check] Build installers without certs"
```

Comment:
Also I would name this a bit differently
```suggestion
name: "[Build] unsigned installers and binaries"
```

Reply:
ok